core/main/src/bootstrap/logging_bootstrap_step.rs

```diff
@@ -0,0 +1,43 @@
+use crate::{state::bootstrap_state::BootstrapState, SEMVER_LIGHTWEIGHT};
+use ripple_sdk::{
+    async_trait::async_trait,
+    framework::{bootstrap::Bootstep, RippleResponse},
+    log,
+    utils::logger::init_and_configure_logger,
+};
+
+pub struct LoggingBootstrapStep;
+
+#[async_trait]
+impl Bootstep<BootstrapState> for LoggingBootstrapStep {
+    fn get_name(&self) -> String {
+        "LoggingBootstrapStep".into()
+    }
+
+    async fn setup(&self, state: BootstrapState) -> RippleResponse {
+        //let device_manifest = &state.platform_state.device_manifest.configuration;
+        let manifest = state.platform_state.get_device_manifest();
+        let log_level = match manifest
```

Comment:
I would rename this to `log_signal_level` just so it's clear it's not the log level in general.